.travis.yml

```diff
@@ -25,7 +26,7 @@ services:
   - docker
 
 before_install:
-  - docker load -i /tmp/docker_images/images.tar || true
+  - docker-compose pull wordpress
```

Comment:
This pulls them in from https://hub.docker.com/repository/docker/xwpco/stream-wordpress